main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Projects.OptionPanels/ProcessRunConfigurationEditor.cs

```diff
@@ -74,18 +74,24 @@ public ProcessRunConfigurationEditorWidget ()
 			mainBox.Margin = 12;
 			var table = new Table ();
 
-			table.Add (new Label (GettextCatalog.GetString ("Arguments:")), 0, 0);
+			var argumentsLabel = new Label (GettextCatalog.GetString ("Arguments:"));
+			table.Add (argumentsLabel, 0, 0);
 			table.Add (argumentsEntry = new TextEntry (), 1, 0, hexpand:true);
+			argumentsEntry.Accessible.LabelWidget = argumentsEntry;
```

Comment:
SetTitleFor?

Reply:
why SetTitleFor?

This should be the label and not the entry itself:
```suggestion
			argumentsEntry.Accessible.LabelWidget = argumentsLabel;
```

Reply:
whoops

Reply:
AtkCocoa doesn't support converting AtkRelationships into Cocoa ones. So we can set the title with
`argumentsEntry.Accessible.SetTitleUIElement (argumentsLabel.Accessible)`